### ISSUE TITLE
Ability to set min/maxsize with %, not just int

### ISSUE THREAD

**Issue by Ligthiago** [hyprwm/Hyprland#5099]
### Description

Considering the fact that the `size` windowrule can be set as a percentage of the screen size, it would be useful to also allow  `minsize` and `maxsize` to be set as percentages.

For example, setting `maxsize 95% 95%` for all windows will restrict floating windows to getting bigger than current screen size and I experience this behavior sometimes, like firefox always do this. It works fine with `int`, if you have similar monitor resolutions and same orientation, but will not work otherwise. 

Or maybe I want to specify, that floating terminal can be minimum 20% size of screen, but if i do math and set it with `int` - it just will dont work as intended with higher resolution monitor. Same with all other use cases, that not require exact pixel perfect size, but must work on all resolutions and screen orientations.

**Tldr:** `maxsize 14% 88%` please, if it possible

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
